### PR TITLE
Add command to populate 'free sms fragment limit'

### PR DESCRIPTION
**Pull request by klssmith**
Added a one-off command to set values of the `free_sms_fragment_limit` in
the `services` table to the default of 250000.

_Note_ - only the last commit has been added - this will be rebased onto master when the branch it is based off is merged.

### CODE REVIEW

File: app/commands.py [hunk 1]
```diff
@@ -305,3 +305,18 @@ def run(self):
         db.session.commit()
 
         print("Populated letter contacts for {} services".format(result.rowcount))
+
+
+class PopulateServiceFreeSmsFragmentLimit(Command):
+
+    def run(self):
+        services_to_update = """
```
Comment (servingUpAces) [inline app/commands.py]: Should we update services_history too?

**Comment by venusbb** [comment]
A default free_sms_fragment_limit parameter is set at the end point /service/{id} where create_service is called. We need to set the default before the dao is called.

File: app/commands.py (hunk 1 above)
Comment (klssmith) [reply app/commands.py]: Now updated to update both tables

**Comment by klssmith** [comment]
Closing, as this change has now been added here instead https://github.com/alphagov/notifications-api/pull/1305

**Pull request closed without merging**